medcat/meta_cat.py

```diff
@@ -157,9 +173,9 @@ def merge_data_loaded(base, other):
         for path in json_path:
             with open(path, 'r') as f:
                 data_loaded = merge_data_loaded(data_loaded, json.load(f))
-        return self.train_raw(data_loaded, save_dir_path)
+        return self.train_raw(data_loaded,save_dir_path,data_=data_)
 
-    def train_raw(self, data_loaded: Dict, save_dir_path: Optional[str] = None) -> Dict:
+    def train_raw(self, data_loaded: Dict, save_dir_path: Optional[str] = None,data_=None) -> Dict:
```

Comment:
this is a confusing method sig now, why is data_ needed alongside data_loaded

Reply:
data_loaded is the actual dataset that we've loaded (from medcat export), data_ is the oversampled additional data that will be added to it the actual data

Reply:
ah okay - `_data` should be renamed to something more descriptive

Reply:
I've replaced it with `data_oversampled`